jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,403 @@
+    "                        max_attempts=0)\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "async with planet.Session() as ps:\n",
+    "    # The Orders API client\n",
+    "    client = planet.OrdersClient(ps)\n",
+    "    # Create the order and deliver it to GCP\n",
+    "    order_details = await create_and_deliver_order(order_params, client)\n",
+    "    "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Basemaps API to get the Basemap and quad IDs\n",
+    "---\n",
+    "\n",
+    "Use the Basemap APIs to retrieve the basemap you want and the quad IDs. You can get the base URL you'll need to communicate with the Basemap API service:\n",
+    "\n",
+    "1. Go to the [Basemap API reference](https://developers.planet.com/docs/basemaps/reference/) at https://developers.planet.com/docs/basemaps/reference/. \n",
+    "2. Under List Mosaics, select GET /mosaics.\n",
+    "\n",
+    "A dropdown UI element appears with the URL to use:\n",
+    "\n",
+    "https://api.planet.com/basemaps/v1/mosaics\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Search for the mosaic ID to get its quad IDs\n",
+    "\n",
+    "Here, we are using the full name of the mosaic, but with the `name__contains` field, you can use a partial name based on the names of the mosaics available through your account.\n",
+    "\n",
+    "Declare the search string for `requests.get()`. You will use the dictionary you create with the mosaic name to pass to the requests.get() function.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth\n",
+    "import json\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth = HTTPBasicAuth(API_KEY, '')\n",
+    "session = requests.Session()\n",
+    "\n",
+    "# Authenticate\n",
+    "session.auth = (API_KEY, \"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASEMAP_API_URL = 'https://api.planet.com/basemaps/v1/mosaics'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "\n",
+    "basemap_params = {\n",
+    "    'name__contains': 'point_reyes_rolling_normalized_2020-04-30_mosaic'\n",
+    "}\n",
+    "\n",
+    "basemap_service_response = requests.get(url=BASEMAP_API_URL, params=basemap_params, auth=auth, headers=headers)\n",
+    "\n",
+    "basemaps = basemap_service_response.raise_for_status()\n",
+    "if basemap_service_response.status_code != 204:\n",
+    "    basemaps = json.loads(basemap_service_response.text)\n",
+    "print(basemaps)\n"
```

Comment:
Get rid of this, please. The within its output is `https://api.planet.com/basemaps/v1/mosaics?api_key=<YOU-API-KEY>`.